apps/dashr-brain-viewer/app.R

```diff
@@ -0,0 +1,455 @@
+                 if (val == 'human'){
+                   data = readRDS('realct.RData')
+                   traces = data
+                   figure = list(
+                     data = list(traces),
+                     layout = plot_layout
+                   )
+                 } else if (val == 'human_atlas'){
+                   data = readRDS('surfreg.RData')
+                   traces = data
+                   figure = list(
+                     data = list(traces),
+                     layout = plot_layout
+                   )
+                 } else if (val == 'mouse'){
+                   data = readRDS('mouse.RData')
+                   traces = data
+                   figure = list(
+                     data = traces,
+                     layout = plot_layout
+                   )
+                 }
+               }
+               
+               if(is.null(clickData[[1]]) == FALSE){
+      
+                   marker = list(
+                     x = list(clickData[['points']][[1]][['x']]),
+                     y = list(clickData[['points']][[1]][['y']]),
+                     z = list(clickData[['points']][[1]][['z']]),
+                     mode = 'markers',
+                     marker = list(size=15, line=list(width=3)),
+                     name = 'Marker',
+                     type = 'scatter3d',
+                     text = 'Click point to remove annotation'
+                   )
+                   
+                   anno = list(list(
+                     x = clickData[['points']][[1]][['x']],
+                     y = clickData[['points']][[1]][['y']],
+                     z = clickData[['points']][[1]][['z']],
+                     font = list(color = 'black'),
+                     bgcolor = 'white',
+                     borderpad = 5,
+                     bordercolor = 'black',
+                     borderwidth = 1,
+                     captureevents = TRUE,
+                     ay = -50,
+                     arrowcolor = 'white',
+                     arrowwidth = 2,
+                     arrowhead = 0,
+                     text = 'Click here to annotate<br>(Click point to remove)'
+                   ))
+                   if(length(figure[['data']]) > 0){
+                     same_point_found = FALSE
+                     for (i in 1:length(figure[['data']])) {
+                       for (point in figure['data']) {
+                         #browser()
+                         if(unlist(point[[i]]['x']) == marker[['x']][[1]] & unlist(point[[i]]['y']) == marker[['y']][[1]] & unlist(point[[i]]['z']) == marker[['z']][[1]]){
+                           ANNO_TRACE_INDEX_OFFSET = 1
+                           if (val == 'mouse'){
+                             ANNO_TRACE_INDEX_OFFSET = 2
+                           }
+                           figure[['data']][i] <- NULL
+                           print(list('DEL. MARKER', i, figure[['layout']][['scene']][['annotations']]))
```

Comment:
this print statement pops up on my console, but not on the app. Maybe it was used for debugging?

Reply:
@KPhans ☝️ is the print statement still present?